src/integrators/aov.cpp

```diff
@@ -184,6 +184,9 @@ class AOVIntegrator final : public SamplingIntegrator<Float, Spectrum> {
 
         if (m_aov_names.empty())
             Log(Warn, "No AOVs were specified!");
+
+        if (m_integrators.empty())
+            Throw("No child integrator specified!");
```

Comment:
It is sometimes useful to use the AOV integrator without a nested integrator, e.g. for quickly looking at shading normals or UV coordinates. Hence I wouldn't enfore the instantiation of a nested integrator.

Reply:
Good point, I've removed this constraint!